CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Kobweb IntelliJ Plugin
 
+## [0.1.4]
```

Comment:
Very minor question: What do you think about 0.2.0? This seems like a bigger change than normal.